### PR TITLE
Adding Seller Management

### PR DESCRIPTION
**Pull request by Parathantl**

### CODE REVIEW

File: src/main/java/com/shopping/shoppingapi/controller/SellerController.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import com.shopping.shoppingapi.model.Seller;
+import com.shopping.shoppingapi.service.SellerServices;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+    @CrossOrigin("*")
```
Comment (rusiruavb) [inline src/main/java/com/shopping/shoppingapi/controller/SellerController.java]: Remove additional tab space

File: src/main/java/com/shopping/shoppingapi/model/Seller.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    private String address_1;
+    private String address_2;
+    private String city;
+    private String country;
+
+    @OneToMany(mappedBy = "seller")
+    @JsonManagedReference
+    private List<Shop> shops;
+
+    public Integer getId() {
```
Comment (rusiruavb) [inline src/main/java/com/shopping/shoppingapi/model/Seller.java]: Remove all getters and setters

**Review by rusiruavb — CHANGES_REQUESTED** [review]
Need to do some changes

File: src/main/java/com/shopping/shoppingapi/controller/SellerController.java (hunk 1 above)
Comment (Parathantl) [reply src/main/java/com/shopping/shoppingapi/controller/SellerController.java]: Done

File: src/main/java/com/shopping/shoppingapi/model/Seller.java (hunk 2 above)
Comment (Parathantl) [reply src/main/java/com/shopping/shoppingapi/model/Seller.java]: Done

**Review by rusiruavb — APPROVED** [review]
Ready to merge

**Pull request merged**